package.json

```diff
@@ -118,6 +118,8 @@
     "is-pull-stream": "~0.0.0",
     "is-stream": "^2.0.0",
     "iso-url": "~0.4.6",
+    "it-pipe": "^1.0.1",
+    "it-glob": "^0.0.1",
```

Comment:
Can't help but mentally exclaim "it pipe!, it glob!" in childlike wonder when I read this.